nautilus_trader/adapters/dydx/execution.py

```diff
@@ -1071,11 +1078,18 @@ async def _submit_order_single(self, order) -> None:
         }
 
         price = 0
+        trigger_price = 0
```

Comment:
Perhaps merge the latest develop branch into your branch. I've made similar changes too but haven't gotten to thoroughly testing it yet.

Reply:
yes we  merged the develop branch in to ours.
we are currently testing it.

Reply:
Great! Thanks.